tests/app/services/user_authentication/test_token_manager.py

```diff
@@ -19,28 +19,17 @@ def test_is_api_key_returns_true_when_audience_has_api_key_entry(self):
 
         assert manager.is_api_key() is True
 
-    def test_refresh_calls_refresh_api_key_method_when_is_api_key_method_returns_true(self, mocker, fake):
+    def test_refresh_calls_refresh_api_key_method_when_is_api_key_method_returns_true(self, requests_mock, fake):
```

Comment:
Please update test name, since it's doesn't call `refresh_api_key` method anymore, rather calls refresh token endpoint.